### PR TITLE
SubsetLoader and support multiple contexts.

### PR DESCRIPTION
**Pull request by tokejepsen**
- SubsetLoader only appears once in the right click menu.
- If a SubsetLoader support multiple contexts with `is_multiple_context_compatible` attribute, it'll get a list of contexts passed to its `load` method.

### CODE REVIEW

**Review by mkolar — APPROVED** [review]

File: avalon/pipeline.py [hunk 1]
```diff
@@ -1313,6 +1335,67 @@ def get_repres_contexts(representation_ids, dbcon=None):
...
+    })
+    asset_docs_by_id = {
+        asset_doc["_id"]: asset_doc
+        for asset_doc in asset_docs
+    }
+
+    project_doc = dbcon.find_one({"type": "project"})
+
+    for id, doc in subset_docs_by_id.items():
+        asset_doc = asset_docs_by_id[subset_doc["parent"]]
```
Comment (iLLiCiTiT) [inline avalon/pipeline.py:1385]: Should be `asset_docs_by_id[doc["parent"]]` or maybe rather change loop variable names `id` -> `subset_id` (as `id` in python builtin function) and `doc` to `subset_doc` to say what it is :)

File: avalon/pipeline.py [hunk 2]
```diff
@@ -1313,6 +1335,67 @@ def get_repres_contexts(representation_ids, dbcon=None):
...
+
+    _subset_ids = []
+    for id in subset_ids:
+        if isinstance(id, six.string_types):
+            id = io.ObjectId(id)
+        _subset_ids.append(id)
+
+    subset_docs = dbcon.find({
+        "type": "subset",
+        "_id": {"$in": list(subset_ids)}
```
Comment (iLLiCiTiT) [inline avalon/pipeline.py:1365]: Misused variable `"subset_ids"` should be `"_subset_ids"`.

File: avalon/pipeline.py (hunk 1 above)
Comment (tokejepsen) [reply avalon/pipeline.py:1385]: Good catch! Fixed.

File: avalon/pipeline.py (hunk 2 above)
Comment (tokejepsen) [reply avalon/pipeline.py]: Good catch! Fixed.

**Pull request merged**